packages/app/configuration/index.ts

```diff
@@ -101,6 +101,12 @@ export async function getConfiguration({
       apiValue: apiConfiguration.mocksHarKeyManager,
       defaultValue: defaultHarKeyManager,
     }),
+    harMimeTypesParseJson: buildProperty<Array<string>>({
+      cliValue: [],
```

Comment:
```suggestion
      cliValue: null,
```
(if it is not null, it will always override any other value, and make it useless to have a global configuration)